test/unit/gateways/braintree_blue_test.rb

```diff
@@ -646,10 +646,16 @@ def test_android_pay_card
     assert_equal "transaction_id", response.authorization
   end
 
-  def test_supports_network_tokenization
+  def test_supports_network_tokenization_true
+    @gateway.stubs(supports_network_tokenization_brands: ['visa'])
```

Comment:
Shouldn't you be stubbing `Braintree::MerchantGateway` rather than a private method?